packages/specification/schemas/requests/dispensing/Withdraw.yaml

```diff
@@ -0,0 +1,90 @@
+type: object
+properties:
+  resourceType:
+    type: string
+    enum: [Task]
+  id:
+    type: string
+    example: 6a2624a2-321b-470e-91a6-8ae7a065e2f0
+  contained:
+    type: array
+    items:
+      oneOf:
+      - $ref: ../../components/PractitionerRole/TaskPractitionerRole.yaml
+      - $ref: ../../components/Organization/DispenseNotificationOrganization.yaml
+  extension:
+    type: array
+    description: |
+      FHIR extension array to hold Repeat Information.
+      This identifies the instance of the eRD prescription that is being returned.
+      Given that all eRD prescriptions have the same prescription ID,
+      the combination of Prescription ID and repeat number identifies the issue to withdraw or return.
+    items:
+      $ref: ../../components/_extensions/EPSRepeatInformation.yaml
+  identifier:
+    description: The unique identifier for the Withdraw Request.
+    type: array
+    items:
+      $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/Rfc4122
+  status: 
+    type: string
+    example: in-progress
+  statusReason:
+    type: object
+    description: The reason the prescription is being withdrawn, which must be coded as per the Withdraw Reason vocabulary.
+    properties:
+      coding: 
+        type: array
+        items:
+          $ref: ../../components/_fragments/codes.yaml#/components/schemas/EpsTaskDispenseWithdrawStatusReason
+  intent:
+    type: string
+    description: The intent of the Medication Requests being withdrawn.
+    example: order
+  code: 
```

Comment:
Not sure what this is, can't see it being used in our code.

Reply:
Task-code I can currently see being used in task-validator.ts and translation.ts, unless you're referring to the Intent which I can only see being forced to TaskIntent.ORDER in translation, and anything else fails the task-validator

Reply:
Ah yes, again we should probably inline this and explicitly state that it must be "abort".

Reply:
Will do